src/mem/Pool.h

```diff
@@ -166,9 +166,14 @@ class MemPools
     void setDefaultPoolChunking(bool const &);
 
     MemImplementingAllocator *pools = nullptr;
-    ssize_t mem_idle_limit = (2 << 20) /* 2MB */;
     int poolCount = 0;
     bool defaultIsChunked = false;
+
+private:
+    /// Limit for allocated (but unused) memory in memory pools.
+    /// Initial value is 2MB until first configuration,
+    /// \see squid.conf memory_pools_limit directive.
```

Comment:
I hope this `\see` reference emits Doxygen errors or warnings. Let's avoid Doxygen commands with parameters that Doxygen cannot grok:

```suggestion
    /// See squid.conf memory_pools_limit directive.
```